src/dotty/tools/dotc/Compiler.scala

```diff
@@ -41,56 +41,58 @@ class Compiler {
    */
   def phases: List[List[Phase]] =
     List(
-      List(new FrontEnd),           // Compiler frontend: scanner, parser, namer, typer
```

Comment:
I'd prefer not to loose git history here.
Indentation is not that important.

Additionally, it will introduce a lot of merge conflicts with linker.